packages/optimizer/README.md

```diff
@@ -1,115 +1,121 @@
-# AMP Optimizer
+# AMP Optimizer 
 
 [![npm version](https://badge.fury.io/js/amp-toolbox-optimizer.svg)](https://badge.fury.io/js/amp-toolbox-optimizer)
 
-AMP Optimizer - the performance optimizations of the AMP Cache on your own site.
+AMP Optimizer is a tool to server-side enhance the rendering performance of AMP pages. AMP Optimizer implements [AMP performance best practices](https://amp.dev/documentation/guides-and-tutorials/optimize-and-measure/optimize_amp?format=websites) and supports [AMP server-side-rendering](https://amp.dev/documentation/guides-and-tutorials/optimize-and-measure/server-side-rendering?format=websites). By default, it will perform the following optimizations:
```

Comment:
I think it's worth mentioning that the optimizer generates valid AMP (given valid input) somewhere here for clarity (even if briefly) especially since this is a change from previous versions.

Reply:
good point! Done!